cmake/faceliftMacros.cmake

```diff
@@ -596,7 +596,7 @@ endfunction()
 function(facelift_add_qml_plugin PLUGIN_NAME)
 
     set(options )
-    set(oneValueArgs URI VERSION)
+    set(oneValueArgs URI VERSION IMPORT_FOLDER)
```

Comment:
well, in Qt plugins exported to QML are usually placed in a folder called "import" - same here, so it would be rather misleading to name this EXPORT_FOLDER, wouldn't it?